yarn-project/circuits.js/src/scripts/constants.in.ts

```diff
@@ -333,6 +333,8 @@ function parseNoirFile(fileContent: string): ParsedContent {
 function evaluateExpressions(expressions: [string, string][]): { [key: string]: string } {
   const constants: { [key: string]: string } = {};
 
+  const knownBigInts = ['AZTEC_EPOCH_DURATION', 'FEE_RECIPIENT_LENGTH'];
```

Comment:
Why do we need these to be bigints?

Reply:
We don't, but the generated constants don't come out as bigint in the TS, so it seems fine.